apps/mobile/src/features/token/components/token-price-change.tsx

```diff
@@ -22,40 +21,35 @@ export function TokenPriceChange({
   price: Money;
   changePercent: number;
 }) {
-  const { i18n } = useLingui();
   const priceAmount = typeof price.amount === 'number' ? price.amount : Number(price.amount);
-
   const priceChange = (priceAmount * changePercent) / 100;
   const priceChangeFiat = createMoney(priceChange, price.symbol);
+  const color = getPriceChangeColor(changePercent);
 
   return (
-    <>
+    <Box flexDirection="row" alignItems="baseline" gap="1">
       {changePercent !== 0 && (
         <ArrowTriangleTopIcon
-          color={getPriceChangeColor(changePercent)}
+          color={color}
           width={8}
           height={8}
-          style={{ transform: [{ rotate: changePercent < 0 ? '180deg' : '0deg' }] }}
+          style={{
+            alignSelf: 'center',
+            transform: [{ rotate: changePercent < 0 ? '180deg' : '0deg' }],
+          }}
         />
       )}
-      <Text variant="label02" color={getPriceChangeColor(changePercent)}>
-        {i18n._({
-          id: 'token.details.price_change_percentage',
-          message: '{changePercent}%',
-          values: {
-            changePercent: changePercent.toFixed(2),
-          },
-        })}{' '}
-        (
-        {/* FIXME LEA-3015: this is showing an operator for negative values (-), but we don't want that in the design */}
-        <Balance
-          balance={priceChangeFiat}
-          variant="label02"
-          lineHeight={16}
-          color={getPriceChangeColor(changePercent)}
-        />
-        )
+      <Text variant="label02" color={color}>
+        {`${changePercent}% `}
```

Comment:
This is a great example of how much better and simpler this new implementation is. Great work 👏